systemtest/src/test/java/io/strimzi/systemtest/kafka/listeners/ListenersST.java

```diff
@@ -2155,6 +2163,113 @@ void testMessagesTlsScramShaWithPredefinedPassword(ExtensionContext extensionCon
         );
     }
 
+    @ParallelNamespaceTest
+    @Tag(NODEPORT_SUPPORTED)
```

Comment:
You are not really connecting to it. So I think you don't needs this. the nodeports are always supported by Kube ... just sometimes you cannot connect to them.